### PR TITLE
8.0 account_credit_control: small enhancements

### PR DESCRIPTION
**Pull request by alexis-via**
Fix integer field definition
Automatically add admin and demo user to credit control groups, for easier testing

### CODE REVIEW

**Comment by guewen** [comment]
:+1:

**Comment by adrienpeiffer** [comment]
:+1:

File: account_credit_control/line.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ class CreditControlLine(models.Model):
                                 store=True,
                                 readonly=True)
 
-    level = fields.Integer('credit.control.policy.level',
+    level = fields.Integer(string='Policy Level',
```
Comment (pedrobaeza) [inline account_credit_control/line.py:133]: Maybe we should add it to the translation template, or add Transifex Travis job...

**Comment by alexis-via** [comment]
Could someone with the super-powers merge this small change ?

I didn't modify the POT file because I was told that we shouldn't do that on projets that manage the translation with Transiflex.

**Comment by sebastienbeau** [comment]
Transiflex will do the job for us

**Pull request merged**